src/Routes.js

```diff
@@ -75,6 +76,7 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/chair_console" component={ChairConsole} props={props} rolesAllowed={[USER_ROLES.chairperson]} />
     <AuthenticatedRoute path="/new_chair_console" component={NewChairConsole} props={props} rolesAllowed={[USER_ROLES.chairperson]} />
     <AuthenticatedRoute path="/member_console" component={MemberConsole} props={props} rolesAllowed={[USER_ROLES.member]} />
+    <AuthenticatedRoute path="/new_member_console" component={NewMemberConsole} props={props} rolesAllowed={[USER_ROLES.member]} />
```

Comment:
We should just bite the bullet here and make the path to `member_console` point to `NewMemberConsole`. A follow on PR could replace `MemberConsole` with `NewMemberConsole`.